.travis.yml

```diff
@@ -1,5 +0,0 @@
-language: smalltalk
-sudo: false
-
-smalltalk:
-  - Squeak-trunk
```

Comment:
We need this for coverage reporting. (See above)

Reply:
what you guys need... ;D